benchmarks/histogram_other_test.go

```diff
@@ -3,8 +3,9 @@ package ddd
 import (
 	"testing"
 
+	"github.com/HdrHistogram/hdrhistogram-go"
```

Comment:
Is this import used?

Reply:
Looks like it's this one:

https://github.com/signalsciences/dogdirect/blob/main/benchmarks/histogram_other_test.go#L29

https://github.com/HdrHistogram/hdrhistogram-go/blob/master/hdr.go#L83